content/posts/newsletter/2020-07-01.md

```diff
@@ -0,0 +1,168 @@
+
+![profanity 0.9 screenshots](/images/newsletter/june2020/profanity090.png "profanity 0.9") 
+
+[JSXC](https://jsxc.org) released [version 4.1.1](https://github.com/jsxc/jsxc/releases/tag/v4.1.1) which fixes several video related issues and enhances interoperability with Conversations. Additionally, a [first experimental desktop build](https://www.jsxc.org/blog/2020/06/25/JSXC-Desktop.html) for all platforms was released. The project is looking for early testers and people with experience in Electron and the Travis build chain.
+
+![Desktop client JSXC](/images/newsletter/june2020/jsxc41.png "Desktop client JSXC")
+
+Pix-Art Messenger removed their [OTR support by the 30th of June](https://social.tchncs.de/@pixart/104349576991457471). The general note on this can be [read on Github](https://github.com/kriztan/Pix-Art-Messenger/blob/master/docs/encryption.md).
+
+And last but not least - can you believe this? [Pidgin 2.14.0](https://pidgin.im/posts/2020-06-2.14.0-released/) 😯 There is also a live development [Twitch stream](https://twitter.com/rw_grim/status/1278857182629498880) every Thursday.
+
+### Servers
+
+The ignite realtime community released [Openfire Beta 4.6.0](https://discourse.igniterealtime.org/t/openfire-4-6-0-beta-is-released/88203), implementing  XEP-0289, Federated MUC for Constrained Environments and improved PEP/PubSub support. If you have time to test please join them to help out!
+
+## Libraries
+
+Gajim’s XMPP library '[python-nbxmpp](https://dev.gajim.org/gajim/python-nbxmpp/-/tree/master)' had its 1.0 release. Being a friendly fork of the xmppy library back in the days, many things changed since then. A lot of work went into it, especially during the past year. If you’re interested, there is a simple example client included.
+
+## Other
+
+Did you actually know? [League of Legends uses XMPP in their backend](https://technology.riotgames.com/news/chat-service-architecture-protocol).
+
+## Google Summer of Code
+
+Since the beginning of May students all over the world are working on many open-source projects within the Google Summer of Code Season 2020. We would like to introduce the students who work on [GSoC XMPP projects](https://summerofcode.withgoogle.com/organizations/5474262808264704/#6018598289539072) and share their blog posts:
+
+[Aditya Borikar](https://adiaholic.github.io/gsoc2020/2020/05/16/Chapter-0-Introduction.html) is working on WebSocket support for Smack. Blog history: 
```

Comment:
Markdown requires a blank line after this line for the following ones to be parsed as a list.

Reply:
Thank you Zash, we will use this in the upcoming version!